packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -673,13 +673,6 @@ export class RichText extends Component {
 		return value;
 	}
 
-	componentWillReceiveProps( nextProps ) {
```

Comment:
I've merged master and resolved a conflict. `componentWillReceiveProps` was removed after I diverged from master: https://github.com/WordPress/gutenberg/commit/27a6ead99b6ec38106c2b8696c74fe2a6e3d24d2#diff-4828a21853e899e5a36faecfa96d83e8L613.